src/frontend/scss/settings/_colors.scss

```diff
@@ -437,7 +437,7 @@ $r-theme: (
     pubdate-border: r-color('firebrick6'),
     authors-border: r-color('firebrick6'),
     cover-empty-background: r-color('smoke'),
-    body-color: transparent,
+    body-color: r-color('black'),
```

Comment:
I imagine this to cover case when content have not any height to fill the browser height, why not.

Reply:
the body of blog posts was white on white and not visible that's why I made this change.

Reply:
Another bug i didn't see, i just have to use "inherit"